### ISSUE TITLE
Inconsistent behaviour of KeggParser

### ISSUE THREAD

**Issue by EmanuelGoncalves** [cokelaer/bioservices#30]
Hi,

KeggParser behaves differently for the same attribute in different entries, for instance if a compound has only one name KeggParser returns a string, although if it has more than one it returns a list. This makes it hard to automatically parse all the compounds names.

I would suggest that attributes that can be lists should always be returned as lists independently of the number of elements found. This happens with other attributes, e.g. reactions.

Thanks.

Cheers,

**Comment by cokelaer** [comment]
Fixes in revision b2b02fe..c25ed1c

**Comment by cokelaer** [comment]
This was non trivial. I took this opportunity to revisit the way parsing is made. This is not completely finalised but the latest version should handled the different caes in a more transparent way. Note however, that the API of thedatabase itself is not always consistent. 

Reactions are either transformed into a list if it looks like a list of reactions or a dictionaries if reactions are followed by a description. 

I will detail the different cases in the documentation in the coming days.

**Comment by cokelaer** [comment]
New version of bioservices have been uploaded on pypi with major revision on how the KEGG parser handles the entries. This should be more robust now. Please use version 1.3.6 from pypi or the source code. 

The API is now as follows . There is no need to use KEGGParser class anymore, just use KEGG itself and its parse() method
```
from bioservices import KEGG
k = KEGG()
k.parse(k.get('hsa:7535'))
```
If the entry can be interpreted, a dictionary is returned. Otherwise the raw data is returned. So a string means the parsing did not work (no error returned anymore). This new release also fixes the previous bug report #29
